lib/inferno/entities/test.rb

```diff
@@ -106,11 +106,13 @@ class << self
         # @return [void]
         # @example
         #   input :patient_id, :bearer_token
-        def input(*input_definitions)
+        def input(key, *other_keys, **input_definitions)
```

Comment:
I think it might be best to ditch taking a plain list of inputs. We should probably have a chat with Rob and get his opinion.

Reply:
Not sure if we talked about this. I'm pretty ambivalent about this so I'm okay with removing it.